misc/net.minetest.minetest.desktop

```diff
@@ -1,13 +1,13 @@
 [Desktop Entry]
 Name=Minetest
 GenericName=Minetest
-Comment=Multiplayer infinite-world block sandbox
-Comment[de]=Mehrspieler-Sandkastenspiel mit unendlichen Blockwelten
-Comment[es]=Juego sandbox multijugador con mundos infinitos
-Comment[fr]=Jeu multijoueurs de type bac à sable avec des mondes infinis
-Comment[ja]=マルチプレイに対応した、無限の世界のブロック型サンドボックスゲームです
-Comment[ru]=Игра-песочница с безграничным миром, состоящим из блоков
-Comment[tr]=Tek-Çok oyuncuyla küplerden sonsuz dünyalar inşa et
+Comment=Block-based multiplayer game platform
+Comment[de]=Blockbasierte Multiplayer-Spielplattform
+Comment[es]=Plataforma de juegos multijugador basada en bloques
+Comment[fr]=Plate-forme de jeu multijoueurs à base de blocs
+Comment[ja]=ブロックベースのマルチプレイヤーゲームプラットフォーム
```

Comment:
are these google translate?

Reply:
Outside of french, yes (actually DeepL).

If some people with knowledge of these languages could check that would be great.

Reply:
The German one unnecessarily uses an anglicism and while the Japanese one is correct (afaict) it's a loanword soup.

I'd rather remove these and wait for native speakers to add correct translations.